control_software/src/snap_fengine.py

```diff
@@ -17,9 +17,11 @@
 PAM_MIN = 0
 
 
-def _jsonify(val, cast=True):
+def _jsonify(var, val, cast=True, debug=False):
```

Comment:
As far as I can see, adding var here is only for debug to print with. I don't think that usage warrants requiring all _jsonify calls to report a variable name, and passing the name as an argument strongly implies that it is used somehow in the jsonification process. Suggest reverting.

Reply:
I've reverted.